avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -304,6 +304,12 @@ particleProperties =
 writePartToCSV = False
 # particle properties to be saved to csv (options: ux, uy, uz, velocityMagnitude,..)
 particleProperties = velocityMagnitude|m
+# create range-time diagram of simulation results -
+# additional configurations in ana5Utils - distanceTimeAnalysisCfg
+createRangeTimeDiagram = False
+# if TTdiagram=True - create thalweg-time diagram,
+# if False create range-time diagram with radar field of view
+TTdiagram = True
```

Comment:
I'm really against making/controlling this from a INI file! This is visualisation and should be done AFTER the model. Especially since we want to have it applicable to more models.
If someone wants to plott the TTdiag, they run the model and THEN do the analysis /plotting. This Section of the config should contain stuff that might be needed FOR visualisation, but it should not trigger the analysis / visualisation

Reply:
changed this to just produce the mtiInfo and save this to pickle and then it is available